canonical-websites/www.ubuntu.com
# Privacy notice update

## Done

- Check that the paragraphs have been updated, just like the last PR on master

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/dataprivacy](http://0.0.0.0:8001/legal/dataprivacy)
- Check the links work and the cherry pick was successful


## Issue / Card

Fixes: #3700
